source/github-helpers/selectors.ts

```diff
@@ -44,6 +44,11 @@ export const prCommitStatusIcon_ = [
 	'https://github.com/refined-github/sandbox/pull/10',
 ];
 
+export const prOpenOrDraftRow = '.js-issue-row:has(.octicon-git-pull-request.color-fg-open,.octicon-git-pull-request-draft)';
+export const prOpenOrDraftRow_ = [
+	'https://github.com/refined-github/sandbox/issues?q=conflict',
+];
```

Comment:
IDK my local test is so many breaking. It is not verified.

Reply:
Connection issues probably. CI is green so it's all good